plugins/a11yhelp/plugin.js

```diff
@@ -20,32 +20,190 @@
+		 * @since 4.8.0
+		 * @private
+		 * @param {CKEDITOR.editor} editor Editor instance
+		 * @returns {Object}
+		 */
+		_createKeyMap: ( function() {
+			var keyMapCache;
+			return function( editor ) {
+				if ( !keyMapCache ) {
+					var lang = editor.lang.a11yhelp,
+						coreLang = editor.lang.common.keyboard;
+
+					keyMapCache = {
+						8: coreLang[ 8 ],
+						9: lang.tab,
+						13: coreLang[ 13 ],
+						16: coreLang[ 16 ],
+						17: coreLang[ 17 ],
+						18: coreLang[ 18 ],
+						19: lang.pause,
+						20: lang.capslock,
+						27: lang.escape,
+						33: lang.pageUp,
+						34: lang.pageDown,
+						35: coreLang[ 35 ],
+						36: coreLang[ 36 ],
+						37: lang.leftArrow,
+						38: lang.upArrow,
+						39: lang.rightArrow,
+						40: lang.downArrow,
+						45: lang.insert,
+						46: coreLang[ 46 ],
+						91: lang.leftWindowKey,
+						92: lang.rightWindowKey,
+						93: lang.selectKey,
+						96: lang.numpad0,
+						97: lang.numpad1,
+						98: lang.numpad2,
+						99: lang.numpad3,
+						100: lang.numpad4,
+						101: lang.numpad5,
+						102: lang.numpad6,
+						103: lang.numpad7,
+						104: lang.numpad8,
+						105: lang.numpad9,
+						106: lang.multiply,
+						107: lang.add,
+						109: lang.subtract,
+						110: lang.decimalPoint,
+						111: lang.divide,
+						112: lang.f1,
+						113: lang.f2,
+						114: lang.f3,
+						115: lang.f4,
+						116: lang.f5,
+						117: lang.f6,
+						118: lang.f7,
+						119: lang.f8,
+						120: lang.f9,
+						121: lang.f10,
+						122: lang.f11,
+						123: lang.f12,
+						144: lang.numLock,
+						145: lang.scrollLock,
+						186: lang.semiColon,
+						187: lang.equalSign,
+						188: lang.comma,
+						189: lang.dash,
+						190: lang.period,
+						191: lang.forwardSlash,
+						192: lang.graveAccent,
+						219: lang.openBracket,
+						220: lang.backSlash,
+						221: lang.closeBracket,
+						222: lang.singleQuote
+					};
+					keyMapCache[ CKEDITOR.ALT ] = coreLang[ 18 ];
+					keyMapCache[ CKEDITOR.SHIFT ] = coreLang[ 16 ];
+					keyMapCache[ CKEDITOR.CTRL ] = CKEDITOR.env.mac ? coreLang[ 224 ] : coreLang[ 17 ];
+				}
+				return keyMapCache;
+			};
+		} )(),
+
+		/**
+		 * Return html which represent keystroke. Result is wrapped with `<kbd></kbd>` tags or other defined by the user.
+		 *
+		 * 		CKEDITOR.plugins.a11yhelp.representKeystroke( CKEDITOR.instances.editor, 4456496 );
+		 * 		"<kbd><kbd>Alt</kbd>+<kbd>0</kbd></kbd>"
+		 *
+		 * @since 4.8.0
+		 * @param {CKEDITOR.editor} editor Editor instance.
+		 * @param {Number} keystroke Number which represent command keystroke.
+		 * @param {String} [wrappingTag=kbd] Wrapping tag. It's inner string of tag: e.g. 'kbd' string in case of `<kbd></kbd>` tags pair.
+		 * Providing empty string will remove wrapping tags from returned result.
+		 * @returns {String} HTML code with key names wrapped with wrapping tag by default it is `<kbd></kbd>`.
+		 */
+		representKeystroke: function( editor, keystroke, wrappingTag ) {
```

Comment:
Please, make it a private property by adding `@private` tag. Technically this method does not belong to this plugin, but it's fine to leave it here as long as it's only used by a11yhelp plugin.

Reply:
As far as I remember it supposed to be available for everybody. To give user possibility to make nicely wrapped keystroke if he overwrite keystroky description.

Reply:
It's not a valid place for this method, if anything it should be placed in general purpose tools. But I don't think we need to care about this now.

If someone will be interested in reusing this logic, he or she will figure check this private method, and file a proper request if needed.

Reply:
Ok this explains the situation. I'll change it to private then.